addons/skin.estuary/xml/Includes_PVR.xml

```diff
@@ -209,6 +209,12 @@
 					<font>font45</font>
 					<label>$INFO[ListItem.ChannelName]</label>
 				</control>
+				<control type="label">
+					<height>262</height>
+					<font>font25</font>
+					<align>right</align>
+					<label>$VAR[FlagLabel]</label>
+				</control>
```

Comment:
this could potentially overlap with the channel name.

in general, we already try to put way too much info on the right hand side of the pvr views.
if you want to add something new, start by removing two other items first ;-)
(recent discussion on this subject: https://github.com/xbmc/xbmc/pull/16731)

Reply:
See above

Reply:
<img width="2163" alt="Screenshot 2020-03-19 at 11 47 43" src="https://user-images.githubusercontent.com/4601187/77064477-74ff6080-69d7-11ea-8b95-6c686160adb5.png">

Reply:
Don't put more and more stuff into the info panel, please. It's just an overview panel. For details we have the info dialog.

Reply:
> in general, we already try to put way too much info on the right hand side of the pvr views.

full ack @ronie

Reply:
Hmmm, Ok, but if I want to see new shows at a glance I must visit the info screen for each recording to do that. I'm not sure that is feasible. 

Even if we remove redundant info from the top text box the flag still looks best here ;)

Reply:
What, if "new" translated to some other language will be as wide as half the info panel? How will this look like? Will I even see the episode title?

Reply:
Fair. Maybe I could remove the date field instead, it's kind of redundant in both this view and for timers.

Reply:
Sounds good.

Reply:
Ok, how's this for the info panel:

<img width="2156" alt="Screenshot 2020-03-19 at 22 48 40" src="https://user-images.githubusercontent.com/4601187/77121759-dce39480-6a33-11ea-9b74-5800a1a6550e.png">

Reply:
How does it look like for recordings and timers?

Reply:
Here is for timers. I didn't capture a recordings example, can get one later when home.

<img width="2161" alt="Screenshot 2020-03-20 at 18 33 36" src="https://user-images.githubusercontent.com/4601187/77195368-6d27e500-6ad9-11ea-8b99-ee19ce1bfad1.png">

Reply:
And recordings:

<img width="2156" alt="Screenshot 2020-03-20 at 20 00 24" src="https://user-images.githubusercontent.com/4601187/77202033-7fa81b80-6ae5-11ea-8d83-d3d4319613f1.png">

Reply:
And recordings with progress bar:

<img width="2165" alt="Screenshot 2020-03-20 at 20 01 38" src="https://user-images.githubusercontent.com/4601187/77202142-aa926f80-6ae5-11ea-8b02-1e40e52ea6fa.png">